### PR TITLE
Fix some compatibility problems for older versions of OS X

### PR DESCRIPTION
**Pull request by tntc**
These fixes should make screen fetch work on older versions of OS X that are supported by [the TigerBrew fork of HomeBrew.](https://github.com/mistydemeo/tigerbrew)

### CODE REVIEW

File: screenfetch-dev [hunk 1]
```diff
@@ -991,7 +991,7 @@ detectcpu () {
 			cpu="Unknown"
 		fi
 	elif [[ "$distro" == "FreeBSD" || "$distro" == "DragonflyBSD" ]]; then
-		cpu=$(sysctl -n hw.model)
+		cpu=$(dmesg | grep CPU: | head -n 1 | sed -r "s/CPU: //" | sed -e 's/([^()]*)//g')
```
Comment (darealshinji) [inline screenfetch-dev]: Why the changes here when this is only for FreeBSD and DragonflyBSD?

Comment (tntc) [inline screenfetch-dev]: Ah! Yes, apologies. That was an update for compatibility  with Raspberry Pis and other ARM devices running FreeBSD.

```
[mscs@ate ~]$ sysctl -n hw.model
sysctl: unknown oid 'hw.model': No such file or directory
[mscs@ate ~]$ dmesg | grep CPU: | head -n 1 | sed -r "s/CPU: //" | sed -e 's/([^()]*)//g'
ARM1176JZ-S rev 7 


[mscs@nyx ~]$ dmesg | grep CPU: | head -n 1 | sed -r "s/CPU: //" | sed -e 's/([^()]*)//g'
AMD FX-8320 Eight-Core Processor            
[mscs@nyx ~]$ sysctl -n hw.model
AMD FX(tm)-8320 Eight-Core Processor           
```

a test on an Intel machine running FreeBSD might be a good idea.  I'll see if I can run one tomorrow.

Comment (tntc) [inline screenfetch-dev]: Ran the intel test:

$ sysctl -n hw.model
Intel(R) Core(TM) i7-4870HQ CPU @ 2.50GHz
$ dmesg | grep CPU: | head -n 1 | sed -r "s/CPU: //" | sed -e 's/([^()]*)//g'
Intel Core i7-4870HQ CPU @ 2.50GHz

**Comment by darealshinji** [comment]
Manually merged in https://github.com/KittyKatt/screenFetch/commit/eb1d3ede23186c94be913f4c5ac1cd13ef5d6c8c and https://github.com/KittyKatt/screenFetch/commit/be80912e5064a73bf3957bde87ccbf86b332f323 .
